### PR TITLE
✅ Add basic Revue tests

### PR DESCRIPTION
**Pull request by PaulAdamDavis**
no issue

The data Revue returns via their API is pretty simple, so these tests are simple.

Covers the a basic post, addPrimaryTag, email, and pubName, removal of UTM properties from link blocks, and Twitter & YouTube embeds.

Alos fixes a linting error:
```sh
79:39  warning  'i' is already declared in the upper scope  no-shadow
```

### CODE REVIEW

File: packages/mg-revue-api/test/processor.test.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+        const post = posts[0];
+
+        post.should.be.an.Object().with.properties('url', 'data');
+
+        post.url.should.eql('https://www.getrevue.co/profile/sampleuser/issues/weekly-newsletter-of-sampleuser-issue-1-123456');
+
+        const data = post.data;
+
+        data.slug.should.eql('https://www.getrevue.co/profile/sampleuser/issues/weekly-newsletter-of-sampleuser-issue-1-123456');
```
Comment (aileen) [inline packages/mg-revue-api/test/processor.test.js:29]: I think `slug` should not include the whole URL, that's why we try to remove it [here](https://github.com/TryGhost/migrate/blob/master/packages/mg-revue-api/lib/processor.js#L180) with regex. Can you check that again?

**Review by aileen — CHANGES_REQUESTED** [review]
Beside my note, it would be cool, if we'd test the removal of UTM properties in links, as done [here](https://github.com/TryGhost/migrate/blob/master/packages/mg-revue-api/lib/processor.js#L9) and also twitter and youtube embeds, as we process [here](https://github.com/TryGhost/migrate/blob/master/packages/mg-revue-api/lib/processor.js#L130-L149)

**Comment by PaulAdamDavis** [comment]
Updated to test Twitter & YouTube embeds, and removal UTM properties.

**Review by aileen — APPROVED** [review]
Awesome stuff!

**Pull request merged**